schema.json

```diff
@@ -183,7 +183,7 @@
         "value": {
           "type": "integer",
           "maximum": 4294967295,
```

Comment:
The types include int64 and uint64, so perhaps these should all be expanded.

Reply:
Done in https://github.com/OpenLightingProject/pid-schema/pull/6/commits/e175ca69a08581d13dcdae5727ec0f11106c8214